app/lib/fake_data/complaints.rb

```diff
@@ -0,0 +1,72 @@
+require "ffaker"
+
+module FakeData::Complaints
+  def self.complaint_data
+    {
+      id: FFaker::Random.rand(9999),
+      attributes: {
+        issueLastUpdated: FFaker::Time.date,
+        creationDate: FFaker::Time.date,
+        closedDate: FFaker::Time.date,
+        reopenedDate: FFaker::Time.date,
+        initialContactDate: FFaker::Time.date,
+        type: FFaker::Random.rand(2),
+        otherType: FFaker::Lorem.word,
+        issue: FFaker::Lorem.phrase,
+        priority: FFaker::Random.rand(3),
+        status: FFaker::Random.rand(3),
+        dueDate: FFaker::Time.date,
+        # these just sound cool for use as the grantee name :)
+        grantee: FFaker::AddressUK.street_name
+      },
+      relationships: {
+        grantAward: {
+          meta: {
+            id: FFaker::Random.rand(9999)
+          }
+        },
+        grantProgram: {
+          meta: {
+            id: FFaker::Random.rand(999)
+          }
+        },
+        region: {
+          meta: {
+            id: FFaker::Random.rand(10)
+          }
+        }
+      },
+      links: {
+        self: "TODO",
+        html: "TODO"
+      }
+    }
+  end
+
+  def self.generate_response
+    json = jsonapi_wrapper
```

Comment:
[thought/nonblocking] if we were leaving this in for longer, i would suggest making a class because the fact that we're creating a wrapper object here and then applying changes to it and then returning it suggests the need to me. (also a bunch of related `self` methods is always a bit of a smell to me.) 

since it's a short-lived one-off, it's fine how it is, but i wanted to raise it just as something to think about for the future!

Reply:
Yeah, that's fair.  I had started making a class at first when I wanted to pass in pagination / filters and then decided against doing too much for a little fake data.  We will probably want to rethink the structure anyway if we want to mock a single complaint in the near future, since we would want to reuse the `data` function.